index.bs

```diff
@@ -828,6 +829,9 @@ partial interface MLGraphBuilder {
                 "ihwo":
                     - [input_channels/groups, height, width, output_channels]
 
+                "iohw":
+                    - [input_channels/groups, output_channels, height, width]
+
```

Comment:
When the `transpose` flag is set to true, "iohw" filter layout should mean `[input_channels, output_channels/groups, height, width]` right? Because transposed convolution up-samples the output as opposed to convolution, which down-samples it. How do we distinguish in the API that the interpretation of "iohw" depends on whether it is a transposed operation? Is this actually the reason to split transposed convolution to its own operations i.e. `convTranspose2d`?

Reply:
Yeah, guys the more I think about this, the more I feel we need a separate `convTranspose2d` to simplify the two different use cases between convolution (downsampling) and transposed convolution (upsampling). It has real implication how we support the known filter layouts. @huningxin What do you think? If you agree, would you like to take this PR a step further and split the operation?

Reply:
Make sense to me too to split it.

Reply:
> @huningxin What do you think? If you agree, would you like to take this PR a step further and split the operation?

SGTM. I'll take that. Thanks @wchao1115 @gramalingam for your feedbacks.